src/pages/Home.js

```diff
@@ -57,18 +77,59 @@ const Home = () => {
                         params: p
                 })
                         .then(function (response) {
+
+                                var content_type;
```

Comment:
```suggestion
                                let content_type;
```
スコープ外参照をしないのであれば無駄に`var`にしない